### DISCUSSION TITLE
My College Name Not In List

### DISCUSSION THREAD

**Discussion by abhishek7477** [community/community#35218] [GitHub Education]
Hii everyone
I recently known about GitHub Education but when I gona to try it i noticed that there is no college listed in our area but all of schools are listed
So i put my College Name manually and proceed by my account not approved. 
Now any one has idea how to avail this benefit I am a student of a Local Govt. College but it's not in a list 
Our college university is JIWAJI UNIVERSITY,  GWALIOR
@github-staff

**Comment by samus-aran** [comment]
Hi @abhishek7477  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
